robot/basestation/static/js/elements/feed/connection-feed.js

```diff
@@ -0,0 +1,5 @@
+$(document).ready(() => {
+
+	var connected = 
+
+})
```

Comment:
what is this.

Please remove this file lol

Reply:
removed